### PR TITLE
Clown Psychedelic Suit

### PR DESCRIPTION
**Pull request by GermanSanta**
Have you ever had a project that drove you insane? Insanity is spending 4+ hours spriting this definition of insanity clown suit. Adding up to a grand total of 336 sprites.

192 in-hand sprites
24 Floor Sprites 
24 uniform sprites
24 uniform fat sprites 
24 backpack sprites 
24 Mob Mask Sprites
24 Shoe sprites

Then coding this all in neatly into the contraband autodrobe. 

Special Thanks to Vonadi for donating 55 sprites. In spriting, the fat psychedelic clown suit, giggles neon honkington, and giggles neon honkington's floor sprite. 
gYIutUC_RziNCswH4kBPYg

Can't thank you enough for saving me all that work buddy.

:cl:
 * rscadd: Added Psychedelic Clown Suit and Fat Suit, Shoes, Backpack, and Mask
 * rscadd: Added Clown Psychedelic Suit Outfit Box to the Autodrobe

### CODE REVIEW

**Comment by SonixApache** [comment]
madman

**Comment by Zth--** [comment]
Good job. Post a gif so we can see it in action

**Comment by GermanSanta** [comment]
Clowngif

**Comment by GermanSanta** [comment]
clowngif

**Comment by boy2mantwicethefam** [comment]
Visual earrape

**Comment by ShiftyRail** [comment]
It's content and therefore inherently good
@boy2mantwicethefam that'd be eyerape

**Comment by brndd** [comment]
I have synesthesia and this is definitely raping my ears.

**Comment by brndd** [comment]
Also, I hope I never see this in game but I'm updooting anyway.

**Comment by N3X15** [comment]
why

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -791,4 +792,13 @@
 	_color = "skelesuit"
 	species_fit = list(VOX_SHAPED)
 	clothing_flags = ONESIZEFITSALL
-	body_parts_covered = FULL_HEAD|BEARD|HIDEHAIR
\ No newline at end of file
+	flags = HIDEBEARDHAIR
```
Comment (Zth--) [inline code/modules/clothing/under/miscellaneous.dm]: Remove this line

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -791,4 +792,13 @@
...
+	body_parts_covered = FULL_HEAD|BEARD
```
Comment (Zth--) [inline code/modules/clothing/under/miscellaneous.dm]: Remove this change, leave it as `body_parts_covered = FULL_HEAD|BEARD|HIDEHAIR`

**Review by Zth-- — CHANGES_REQUESTED** [review]

**Review by Zth-- — APPROVED** [review]

**Comment by FudgePucker** [comment]
Change log

**Comment by Zth--** [comment]
In my opinion the animation's speed should be tuned down, it's way too harsh at its current state

**Comment by GermanSanta** [comment]
Well you're not going to believe this one but Vonadi's current rate was twice as fast as it is now. The current animation speed is the same of that as the default psychedelic suit

**Pull request merged**